src/commands/sfpowerkit/org/connectedapp/create.ts

```diff
@@ -80,7 +81,7 @@ export default class Create extends SfdxCommand {
   // Set this to true if your command requires a project workspace; 'requiresProject' is false by default
   //protected static requiresProject = true;
 
-  public async run(): Promise<AnyJson> {
+  public async execute(): Promise<AnyJson> {
     rimraf.sync("temp_sfpowerkit");
     SFPowerkit.setLogLevel(this.flags.loglevel, this.flags.json);
```

Comment:
All references to `SFPowerkit.setLogLevel()` at the command layer should be removed and replaced by a single setLogLevel in the base class `SFPowerkitCommand`

Reply:
I removed all the SFPowerkit.setLogLevel() references on each of the commands into the base class SFPowerkitCommand.